.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+.idea
```

Comment:
this is wrong. You should keep such value in a global ignore file. See https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
Done !